src/Razor/test/Microsoft.AspNetCore.Razor.Test.MvcShim.Version1_X/Microsoft.AspNetCore.Razor.Test.MvcShim.Version1_X.csproj

```diff
@@ -3,8 +3,6 @@
   <PropertyGroup>
     <TargetFrameworks>$(DefaultNetCoreTargetFramework);net461</TargetFrameworks>
     <PreserveCompilationContext>true</PreserveCompilationContext>
-    <!-- Ignore API warnings in shim projects -->
-    <NoWarn>$(NoWarn);CA1822;CA1018;IDE0060</NoWarn>
```

Comment:
I'd like to keep these no-warn for these shim projects, but I can make a separate PR which causes them all to share a `ShimProject.editorconfig` in the spirit of this cleaner system.

Reply:
I already have this in a follow-up branch.